entrypoint.sh

```diff
@@ -189,6 +218,11 @@ if [[ ${_START_DOCKER_SERVICE} == "true" ]]; then
   ${_PREFIX} service docker start
 fi
 
+# Unset configuration environment variables if the flag is set
+if [[ ${_UNSET_CONFIG_VARS} == "true" ]]; then
+  unset_config_vars
```

Comment:
This is going to be problematic

The last section of this is 

```
# Container's command (CMD) execution as runner user


if [[ ${_RUN_AS_ROOT} == "true" ]]; then
  if [[ $(id -u) -eq 0 ]]; then
    "$@"
  else
    echo "ERROR: RUN_AS_ROOT env var is set to true but the user has been overridden and is not running as root, but UID '$(id -u)'"
    exit 1
  fi
else
  if [[ $(id -u) -eq 0 ]]; then
    [[ -n "${_CONFIGURED_ACTIONS_RUNNER_FILES_DIR}" ]] && chown -R runner "${_CONFIGURED_ACTIONS_RUNNER_FILES_DIR}"
    chown -R runner "${_RUNNER_WORKDIR}" /actions-runner
    # The toolcache is not recursively chowned to avoid recursing over prepulated tooling in derived docker images
    chown runner /opt/hostedtoolcache/
    /usr/sbin/gosu runner "$@"
  else
    "$@"
  fi
```

It still uses some of the ones you just uset here.

Reply:
Thank you for your thorough review!

> It still uses some of the ones you just unset here.

The variables used in the last section are only those prefixed with `_`, so I believe there shouldn't be any issues with unsetting the environment variables without prefixes. What are your thoughts on this?